src/test/java/com/puppycrawl/tools/checkstyle/XpathFileGeneratorAuditListenerTest.java

```diff
@@ -306,8 +308,12 @@ private static void verifyOutput(String expected, AuditEvent... events) {
 
         listener.auditFinished(null);
 
-        assertEquals("expected number of flushes", 1, out.flushCount);
-        assertEquals("expected number of closes", 1, out.closeCount);
+        assertWithMessage("Output steam flush count")
```

Comment:
please share example of output if mismatch happen, I need to see role of this message in result message.
Just curious .... if we can avoid it, as it was convenience in junit.

Reply:
Something like this:
```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   XpathFileGeneratorAuditListenerTest.testCorrectOne:170->verifyOutput:313 Output stream flush count
expected: 111
but was : 1
[ERROR]   XpathFileGeneratorAuditListenerTest.testCorrectTwo:201->verifyOutput:313 Output stream flush count
expected: 111
but was : 1
[ERROR]   XpathFileGeneratorAuditListenerTest.testOnlyOneMatching:229->verifyOutput:313 Output stream flush count
expected: 111
but was : 1
[INFO] 
[ERROR] Tests run: 3160, Failures: 3, Errors: 0, Skipped: 0
```

I can change the violation message to `"unexpected number of flushes"` if you think that it is more readable.

Reply:
no, it looks good:
```
verifyOutput:313 Output stream flush count
expected: 111
but was : 1
```